packages/model-runner/src/main.ts

```diff
@@ -75,6 +76,12 @@ async function main() {
     const inputsDir = INPUT_DIR
     const outputsDir = OUTPUT_DIR
     mkdirp.sync(inputsDir)
+
+    fs.writeFileSync(
```

Comment:
Notes for associated control plane updates:
- We currently mount the input directory as read-only. That will have to change if we are overwriting the file here.
- For cleanliness and ease of debugging, we could consider naming the original file `runnerInputFile.json` or similar when it's passed on from the control plane, and then write a new file `inputFile.json` here instead of overwriting the original. Not blocking.